web/services/models.py

```diff
@@ -65,14 +65,16 @@ def notify_report_received(sender, instance, signal, *args, **kwargs):
     :param args: Unused
     :param kwargs: Unused
     """
-    if instance.user is None:
-        return
 
     email = instance.user.email
-    if email == TEST_EMAIL:
-        # Don't send a notification if there was not email provided as we can't
-        # actively do anything about it
-        return
+    if email in TEST_EMAIL:
```

Comment:
The nested ifs are little hard to read. Could we write this as
```python
# Don't send a slack notification if there was not 
# a useful email, stacktrace or text provided 
# as we can't actively do anything about it
for field in (email, instance.textBox, instance.stacktrace):
	if field in TEST_VALUES:
		return
```
and change `TEST_EMAIL` to `TEST_VALUES` as they are no longer single thing?

Reply:
I added this suggestion to my second commit but I'm not sure this is what I want. The nested ifs only blocked a slack notification if all 3 fields were in TEST_VALUES. The for-if suggestion blocks a slack notification if any one of the 3 fields were in TEST_VALUES, which is not my intention.